sectorlisp.S

```diff
@@ -134,6 +145,7 @@ Intern:	push	%cx				# Intern(cx,di): ax
 
 GetChar:xor	%ax,%ax				# GetChar→al:dl
 	int	$0x16				# get keystroke
+	mov	%ax,%bp				# used for READ
```

Comment:
can we xchg here?

Reply:
I tried replacing with xchg, but it made the output buffer show one character late than the user input. I tried moving it around around the `int`s, but it didn't work either.
Since it's only used for a buffer for READ there may be a good place to change it to an xchg in some other place.

Reply:
I see that we still need %ax later.